ports/esp32s2/common-hal/displayio/ParallelBus.c

```diff
@@ -33,35 +33,211 @@
 #include "shared-bindings/digitalio/DigitalInOut.h"
 #include "shared-bindings/microcontroller/__init__.h"
 
+/*
+ *
+ *  Current pin limitations for ESP32-S2 ParallelBus:
+ *   - data0 pin must be byte aligned (data0 pin options: 0, 8, 16 or 24)
+ *
+ */
+
 void common_hal_displayio_parallelbus_construct(displayio_parallelbus_obj_t* self,
     const mcu_pin_obj_t* data0, const mcu_pin_obj_t* command, const mcu_pin_obj_t* chip_select,
     const mcu_pin_obj_t* write, const mcu_pin_obj_t* read, const mcu_pin_obj_t* reset) {
 
-    mp_raise_NotImplementedError(translate("ParallelBus not yet supported"));
+    uint8_t data_pin = data0->number;
+    if ( (data_pin % 8 != 0) && (data_pin >= 32) ) {
```

Comment:
What is the second part of the check doing? Seems like you should only need the first part.